python/tests/core/test_transformation_function_engine.py

```diff
@@ -354,6 +375,53 @@ def testFunction():
         assert fv._features[0].name == "tf_name"
         assert fv._features[1].name == "tf1_name"
 
+    def test_attach_transformation_fn_fv_q_prefix(self, mocker):
```

Comment:
Good to include more test cases similar to https://github.com/logicalclocks/feature-store-api/pull/836/files#diff-89646453f22a4907bc274d7bec320b9b368786a7ae01bb820846d46ab7f583bb